examples/node8-yarn/build.yml

```diff
@@ -0,0 +1,33 @@
+##
+# This is a build container for the project.
```

Comment:
I've been going back and forth about having a dedicated build config. With the current structure, this might be fully addressed with a rig project script/alias using the main api service from docker-compose.yml